### PR TITLE
Hotfix cannot get transaction status

### PR DESCRIPTION
**Pull request by Andrenzo17**
- update the base url
- add base url v1

### CODE REVIEW

**Comment by Andrenzo17** [comment]
kindly help to review too, mas @omarxp and @Xaxxis

**Review by rizdaprasetya — COMMENTED** [review]
@Andrenzo17 
please change version to v2.5.2 now. Because I tried to use v2.5.1 to rollback version but failed on composer

Due to urgency level, for hotfix you don't need to wait for review. If you are sure Unit/Integration Test runs fine & your own test looks fine, please go ahead with hotfix release. But you have to be sure during your test it works.

File: Midtrans/Config.php [hunk 1]
```diff
@@ -66,8 +66,10 @@ class Config
...
     public static $curlOptions = array();
 
-    const SANDBOX_BASE_URL = 'https://api.sandbox.midtrans.com';
-    const PRODUCTION_BASE_URL = 'https://api.midtrans.com';
+    const SANDBOX_BASE_URL_V1 = 'https://api.sandbox.midtrans.com/v1';
+    const PRODUCTION_BASE_URL_V1 = 'https://api.midtrans.com/v1';
+    const SANDBOX_BASE_URL = 'https://api.sandbox.midtrans.com/v2';
+    const PRODUCTION_BASE_URL = 'https://api.midtrans.com/v2';
     const SNAP_SANDBOX_BASE_URL = 'https://app.sandbox.midtrans.com/snap/v1';
     const SNAP_PRODUCTION_BASE_URL = 'https://app.midtrans.com/snap/v1';
```
Comment (rizdaprasetya) [inline Midtrans/Config.php:74]: After releasing hotfix, please improve the implementation:

According to docs https://api-docs.midtrans.com/#https-request base URL does not include /v1 or /v2. So let's follow docs. Base url is without `/v1`.

The additional `/v1, /v2` on url path should be added on each Function that call the HTTP client, like here:
https://github.com/Midtrans/midtrans-go/blob/7889ad6bdcf9e7fd1f5565fd74f2e81e86f53099/coreapi/clientsubscription.go#L115-L121

**Comment by rizdaprasetya** [comment]
Temporary urgent hotfix has been released: https://github.com/Midtrans/midtrans-php/releases/tag/v2.5.1
It revert to last stable version of v2.4.4

So you have more time to properly fix this issue now, let's be more mindful and ensure stability on next release.

**Comment by Andrenzo17** [comment]
Kindly help to review again kak @rizdaprasetya

**Review by rizdaprasetya — APPROVED** [review]
Nice, looks good now. Ran test on my local, also resulted in green. Please go ahead with release @Andrenzo17. Thanks!

**Pull request merged**